### PR TITLE
Some clean ups to the node client

### LINKED ISSUE #47: [Bug]: Creating a topic with parameters from Node returns timeout.
1) Start with a fresh system

```
$ fluvio install cluster --local
```

2) Setup Node Enviroment

```
$ mkdir custom-chat && cd custom-chat && npm init -y && \
npm install typescript ts-node @types/node -D && \
npm install @fluvio/client -S
```

3) create `producer.ts` with the following content

```
import Fluvio, { TopicReplicaParam } from "@fluvio/client";

// Create Fluvio Client Instance
const fluvio = new Fluvio();

// Handle Fluvio errors
process.on('unhandledRejection', error => {
    console.log(error);
});

(async () => {
    // Connect the fluvio cluster;
    await fluvio.connect();

    // Connect to admin interface
    const admin = await fluvio.admin();
    const topic = await admin.findTopic("hello-fluvio");

    if (topic == null) {
        let topicParams: TopicReplicaParam = {
            partitions: 1,
            replicationFactor: 1,
            ignoreRackAssignment: true
        };

       await admin.createTopic("hello-fluvio", topicParams);
    }

})();

```

4) Run script to create topic

```
$ npx ts-node ./producer.ts
requiring platform specific module
[Error: Fluvio socket error]
```

Once the system is in that state, a topic with this name can no longer be added to Fluvio

```
$ fluvio topic create hello-fluvio
Error: 
   0: Fluvio client error
   1: Fluvio socket error
   2: time out in send and request: 3

Location:
   src/cli/src/root_cli.rs:138

Backtrace omitted.
Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

### PR DESCRIPTION
**Pull request by simlay**
This will close #47 and probably more.

### CODE REVIEW

**Review by sehz — CHANGES_REQUESTED** [review]

File: src/admin.rs [hunk 1]
```diff
@@ -210,6 +210,7 @@ impl FluvioAdminJS {
         name: String,
         spec: SpuGroupSpecWrapper,
     ) -> Result<(), FluvioError> {
+        debug!("CREATING A NEW MANAAGED SPU: {:?}", spec.0);
```
[the remark this answers is not in the document]
Comment (simlay) [reply src/admin.rs]: No reason other than to help me see it in the logs.

**Review by sehz — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -36,17 +36,22 @@ jobs:
...
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest]
         rust: [stable]
         node: [ '12', '14' ]
     steps:
       - name: Install Fluvio Local Cluster
-        uses: infinyon/fluvio@master
+        uses: simlay/fluvio@minikube-driver-fix
```
Comment (simlay) [inline .github/workflows/ci.yml:44]: Should probably merge infinyon/fluvio#579 for this.

Comment (sehz) [reply .github/workflows/ci.yml:44]: Can you turn #579 into non draft?

**Comment by sehz** [comment]
is this ready?

**Comment by simlay** [comment]
> is this ready?

Yup! Once 579 is merged and this is updated to use the fluvio composite action. :crossed_fingers: :rocket:

**Pull request merged**